.gitignore

```diff
@@ -1 +1,2 @@
 dist
+.DS_Store
```

Comment:
Ideally something like this is in your own local global gitignore https://stackoverflow.com/a/7335487/2584550